components/containers/partitioned_vector/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+include(HPX_Option)
```

Comment:
Same here, is this the correct module being included?

Reply:
Same as above.